### PR TITLE
[DOCS] New default query for IM rules

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #1185 and #1176.

Previews:
- [Create a detection rule](https://security-docs_1186.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-indicator-rule) 
- [Detections and alerts](https://security-docs_1186.docs-preview.app.elstc.co/guide/en/security/master/detection-engine-overview.html#support-indicator-rules)

### CODE REVIEW

**Review by joepeeples — APPROVED** [review]
LGTM!

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -231,7 +231,7 @@ wildcard expression: `*:*`.
 IMPORTANT: Data in indicator indices must be <<ecs-compliant-reqs, ECS compatible>>, and so it must contain a `@timestamp` field.
 +
 .. *Indicator index query*: The query and filters used to filter the fields from
-the indicator index patterns.
+the indicator index patterns. The default query (`@timestamp > "now-30d"`) checks the indicator index patterns for matched indicators that have been ingested during the past 30 days.
```
Comment (rylnd) [inline docs/detections/rules-ui-create.asciidoc:234]: Minor quibble: `event.ingested` is the field representing ingest time; `@timestamp` captures when the event occurred (e.g. when the event originally occurred). I would maybe update the wording to reflect this definition:
```suggestion
the indicator index patterns. The default query (`@timestamp > "now-30d"`) checks the indicator index patterns for matched indicators that have occurred in the past 30 days.
```

**Review by rylnd — APPROVED** [review]
LGTM! One nit about the definition of `@timestamp`, but otherwise 👍

File: docs/detections/rules-ui-create.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:234]: Thanks for pointing this out @rylnd! Your recommended changes are much more precise than the current wording. I'll add them into the next draft :)

**Review by benironside — APPROVED** [review]
LGTM

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**